opengrok-indexer/src/main/java/org/opengrok/indexer/web/messages/Message.java

```diff
@@ -46,14 +50,20 @@
 
 public class Message implements Comparable<Message> {
 
+    @JsonProperty("tags")
```

Comment:
Are not these annotations redundant? The variable has a proper name so no need for "renaming". Applies similarly below. https://stackoverflow.com/questions/12583638/when-is-the-jsonproperty-property-used-and-what-is-it-used-for